lazyflow/operators/ioOperators/opExportSlot.py

```diff
@@ -262,18 +262,11 @@ def _export_h5n5(self, compress=False):
         # Create and open the hdf5/n5 file
         export_components = PathComponents(self.ExportPath.value)
         try:
-            if os.path.isdir(export_components.externalPath):  # externalPath leads to a n5 file
-                shutil.rmtree(export_components.externalPath)  # n5 is stored as a directory structure
-            else:
-                os.remove(export_components.externalPath)
-        except OSError as ex:
-            # It's okay if the file isn't there.
-            if ex.errno != 2:
-                raise
-        try:
-            with OpStreamingH5N5Reader.get_h5_n5_file(export_components.externalPath, "w") as h5N5File:
+            with OpStreamingH5N5Reader.get_h5_n5_file(export_components.externalPath, mode="a") as h5N5File:
                 # Create a temporary operator to do the work for us
                 opH5N5Writer = OpH5N5WriterBigDataset(parent=self)
+                if export_components.internalPath in h5N5File:
+                    del h5N5File[export_components.internalPath]
```

Comment:
```suggestion
                h5N5File.pop(export_components.internalPath, None)
```

Reply:
I'm afraid `z5py` does not support `pop` :/

Reply:
Oh, that's unfortunate. In this case:
```python
with contextlib.suppress(KeyError):
    del h5N5File[export_components.internalPath]
```

Reply:
do you really find that more readable than `if ... in ... :` ?

Reply:
It's more "Pythonic": https://docs.python.org/3/glossary.html#term-lbyl.

I like this style overall a bit more because it does things in a single "transaction".

In this case it doesn't really matter though, either option is totally fine!

Reply:
Now I'm torn ;)